code-studio/src/js/components/StudioCellEditor.jsx

```diff
@@ -0,0 +1,109 @@
+/**
+ * @overview React component to allow for easy editing and creation of
+ * Studio Cells
+ * @see @cdo/apps/studio/cell
+ */
+/* global React */
+var constants = require('@cdo/apps/studio/constants');
+
+var avatarList = ["dog", "cat", "penguin", "dinosaur", "octopus",
+    "witch", "bat", "bird", "dragon", "squirrel", "wizard", "alien",
+    "ghost", "monster", "robot", "unicorn", "zombie", "knight",
+    "ninja", "pirate", "caveboy", "cavegirl", "princess", "spacebot",
+    "soccergirl", "soccerboy", "tennisgirl", "tennisboy"];
```

Comment:
It is. :(

Unfortunately, this is actually just the avatar list for one specific Studio skin; defined [inside the loadStudio function here](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/studio/skins.js#L1443); I'm not certain there's a good way to pull it out of that function without changing a lot more about the Studio code than I'd like to.

I suppose I should at the very least add an explanatory comment, and probably do a bit to make it more clear that this editor is mildly skin-specific.